doc/research/06_acting_implementation.md

```diff
@@ -0,0 +1,61 @@
+# Requirements and challenges for an acting implementation
+
+**Summary:** On this page you can find the results of the basic research on acting summed up into resulting requirements and function, that were already agreed upon.
+
+---
+
+## Authors
+
+Gabriel Schwald
+
+### Date
+
+20.11.2022
+
+---
+[[TOC]]
+
+This document sums up all functions (regarding acting) already agreed upon in [#24](https://github.com/ll7/paf22/issues/24), that could be implemented in the next sprint.
+
+## Planned basic implementation of the Acting domain
+
+The basic goals of the acting domain can be summed up as follows:
+The vehicle must know it's own _position_, _heading_ and _velocity_.
+With the addition of a provided _trajectory_ it is then able to provide _throttle_ and _steering angle_ outputs.
+These goals lead to the following requirements:
+
+## List of basic functions
+
+- Longitudinal control
+    - PID controller
+- Lateral control
+    - Pure Pursuit controller
+    - Stanley controller
+
+## List of Inputs/Outputs
+
+- Subscribes to:
+    - [nav_msgs/Odometry Message](http://docs.ros.org/en/noetic/api/nav_msgs/html/msg/Odometry.html) : to get the current position and heading
+    - [nav_msgs/Path Message](https://docs.ros.org/en/api/nav_msgs/html/msg/Path.html) : to get the current trajectory
+    - emergency breaking msg : to initiate emergency breaking
+    - speed limit msg : to get the maximum velocity
+- Publishes:
+    - [CarlaEgoVehicleControl.msg](https://carla.readthedocs.io/projects/ros-bridge/en/latest/ros_msgs/#carlaegovehiclecontrolmsg) : to actually control the vehicles throttle, steering, ...
+
+## Challenges
+
+A short list of challenges for the implementation of a basic acting domain and how they these could be tackled based on the requirements mentioned above.
+
+- The vehicle needs to know its own position => [nav_msgs/Odometry Message](http://docs.ros.org/en/noetic/api/nav_msgs/html/msg/Odometry.html) or [GNSS](https://carla.readthedocs.io/en/latest/ref_sensors/#gnss-sensor) sensor
+- The vehicle needs to know its own velocity => can be calculated from last/current position and time
```

Comment:
Is the speedometer (sensor.other.speedometer) more useful?

https://leaderboard.carla.org/#map-track

Reply:
Thanks for the feedback. Speedometer isn't listed in [the list of sensor](https://carla.readthedocs.io/en/latest/ref_sensors/) but should be the obvious choice (fixed in 9970769).